tests/functional/RemoteWebDriverCreateTest.php

```diff
@@ -92,11 +92,15 @@ public function testShouldCreateInstanceFromExistingSessionId()
 
         // Store session ID
         $sessionId = $originalDriver->getSessionID();
+        $isW3cCompliant = $originalDriver->isW3cCompliant();
 
         // Create new RemoteWebDriver instance based on the session ID
-        $this->driver = RemoteWebDriver::createBySessionID($sessionId, $this->serverUrl);
+        $this->driver = RemoteWebDriver::createBySessionID($sessionId, $this->serverUrl, null, null, $isW3cCompliant);
```

Comment:
from what I understand we should have 2 tests for the 2  number of possible arguments

Reply:
This is ensured by having test builds in both JsonWire and W3C mode, isn't it?

Reply:
Yes right ;)